python/src/indices.rs

```diff
@@ -203,15 +213,60 @@ pub fn transform_vectors(
     );
     RT.block_on(
         Some(py),
-        do_transform_vectors(dataset, column, distance_type, ivf_centroids, pq, dst_uri),
+        do_transform_vectors(
+            dataset,
+            column,
+            distance_type,
+            ivf_centroids,
+            pq,
+            dst_uri,
+            fragments,
+        ),
     )?
 }
 
+async fn do_shuffle_transform_vectors(
```

Comment:
Minor nit: Either `do_shuffle_transformed_vectors` or `do_shuffle_vectors`.  Using `transform` makes it sound like we are shuffling and transforming.